template_content/README.md.jinja

```diff
@@ -40,6 +40,67 @@ This project has been generated using AlgoKit. See below for default getting sta
 1. If you update to the latest source code and there are new dependencies you will need to run `algokit bootstrap all` again
 2. Follow step 3 above
 
+{% if use_github_actions -%}
+### Continuous Integration
+
+  This project uses [GitHub actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) to define CI/CD workflows, see the `.github/workflows` folder for details.
```

Comment:
"for details" implies some sort of explanation will be found there?

Reply:
Fixed